src/components/AddBook.js

```diff
@@ -1,14 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from './booksSlice';
+
+function AddNewBook() {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: Date.now(),
+      title,
+      author,
+    };
+    dispatch(addBook(newBook));
+  };
 
-export default function AddBook() {
   return (
     <section>
       <h2>ADD NEW BOOK</h2>
-      <form>
-        <input type="text" placeholder="Book title" />
-        <input type="text" placeholder="Author" />
-        <input type="submit" value="ADD BOOK" />
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
+          placeholder="Book title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <input
+          type="text"
+          placeholder="Book author"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+        />
+        <button type="submit">Add</button>
```

Comment:
- [ ] When I try adding a new book the `addbook` action is dispatched but nothing is added to the state and also nothing appeared on the page, So please fix that.